data/civ2civ3/README.civ2civ3

```diff
@@ -136,6 +131,7 @@ CORE CHANGES:
   wars between them.
 
 OPTIONAL RULES:
```

Comment:
Do we need the colon here? We usually don't use them in titles

Reply:
good catch. i'll remove